README.md

```diff
@@ -2,26 +2,53 @@
 
 [![Actions Status](https://github.com/ruby/openssl/workflows/CI/badge.svg)](https://github.com/ruby/openssl/actions?workflow=CI)
 
+OpenSSL for Ruby provides access to SSL/TLS and general-purpose cryptography
+based on the OpenSSL library.
 
-OpenSSL provides SSL, TLS and general purpose cryptography. It wraps the
-OpenSSL library.
+OpenSSL for Ruby is sometimes referred to as "openssl" (in all lowercase) or
+"Ruby/OpenSSL" for disambiguation.
+
+## Compatibility and maintenance policy
+
+OpenSSL for Ruby is released as a RubyGems gem. At the same time, it is part of
+the standard library of Ruby. This is called a [default gem].
+
+Stable branches of OpenSSL for Ruby will receive security and bug fixes for the
+[lifecycle of their corresponding Ruby branch][Ruby Maintenance Branches].
+
+|Branch|Maintenance status             |Ruby compatibility|OpenSSL compatibility                       |
+|------|-------------------------------|------------------|--------------------------------------------|
+|3.2   |normal maintenance (Ruby 3.3)  |Ruby 2.7+         |OpenSSL 1.0.2-3.1 (current) or LibreSSL 3.1+|
+|3.1   |normal maintenance (Ruby 3.2)  |Ruby 2.6+         |OpenSSL 1.0.2-3.1 (current) or LibreSSL 3.1+|
+|3.0   |normal maintenance (Ruby 3.1)  |Ruby 2.6+         |OpenSSL 1.0.2-3.1 (current) or LibreSSL 3.1+|
+|2.2   |security maintenance (Ruby 3.0)|Ruby 2.3+         |OpenSSL 1.0.1-1.1.1 or LibreSSL 2.9+        |
+|2.1   |end-of-life (Ruby 2.5-2.7)     |Ruby 2.3+         |OpenSSL 1.0.1-1.1.1 or LibreSSL 2.5+        |
+|2.0   |end-of-life (Ruby 2.4)         |Ruby 2.3+         |OpenSSL 0.9.8-1.1.1 or LibreSSL 2.3+        |
+
+[default gem]: https://docs.ruby-lang.org/en/master/standard_library_rdoc.html
+[Ruby Maintenance Branches]: https://www.ruby-lang.org/en/downloads/branches/
 
 ## Installation
 
-The openssl gem is available at [rubygems.org](https://rubygems.org/gems/openssl).
-You can install with:
+> **Note**
+> The openssl gem is included with Ruby by default, but you may wish to upgrade
+> it to a newer version available at
+> [rubygems.org](https://rubygems.org/gems/openssl).
+
+To upgrade it, you can use RubyGems:
 
 ```
 gem install openssl
 ```
 
-You may need to specify the path where OpenSSL is installed.
+In some cases, it may be necessary to specify the path to the installation
+directory of the OpenSSL library.
```

Comment:
Perhaps like this?

```
In some cases, you may need to specify the path to the directory where the OpenSSL library is installed.
```

Reply:
I don't have the vocabulary to explain this, but I believe "a car engine" is grammatical English.

That said, I don't have a preference on this. I just wanted to make it crystal clear which "OpenSSL" it refers to.

Reply:
I am not sure if my suggestion is correct or not. I think you can keep the current sentence.

Reply:
I think both sentences are correct from the grammatical perspective. I didn't update this sentence, but I don't have an opinion either.

Reply:
Sure, that's all right.